CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-## [15.0.0] - 2019-11-12
+## TBD - TBD
```

Comment:
Should we consider this a 15.1.0 release? It's somewhere between a bug fix and a feature...

Reply:
ah yes forgot to bring that up. I'm for 15.1.0 as well.